core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowFlinkPlugin.scala

```diff
@@ -25,30 +25,20 @@ import spray.json._
 import cloudflow.sbt.CloudflowKeys._
 import cloudflow.blueprint.StreamletDescriptorFormat._
 import cloudflow.sbt.CloudflowKeys._
+import CloudflowBasePlugin._
 
-object CloudflowFlinkPlugin extends CloudflowBasePlugin {
+object CloudflowFlinkPlugin extends AutoPlugin {
   final val flinkVersion                  = "1.9.2"
```

Comment:
constant = Captilize first letter ?

Reply:
There are a few in the Flink plugin also.

Reply:
thanks for pointing out - will change

Reply:
I'm slightly worried about the large number of places where we hardcode these version Strings but I think that is a separate issue to look into and should not block this PR. Something to think about though...